### PR TITLE
jcpicker: Add version 5.6

### PR DESCRIPTION
**Pull request by rashil2000**
Supercedes #6873

### CODE REVIEW

File: bucket/jcpicker.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "version": "5.6",
+    "description": "Color picker and color editor",
+    "homepage": "https://annystudio.com/software/colorpicker/",
+    "license": "Freeware",
```
Comment (pratikpc) [inline bucket/jcpicker.json:5]: Could you also add a URL for the license?

File: bucket/jcpicker.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ],
+    "persist": [
+        "jcpicker.ini",
+        "jcpicker.txt"
+    ],
+    "checkver": "Latest version:</b> ([\\d.]+)",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://annystudio.com/jcpicker.exe"
```
Comment (pratikpc) [inline bucket/jcpicker.json:39]: Any possibility of versioned download URLs?

Comment (rashil2000) [reply bucket/jcpicker.json:39]: I don't see any

Comment (pratikpc) [reply bucket/jcpicker.json:39]: They share hashes?

Comment (rashil2000) [reply bucket/jcpicker.json:39]: No

**Pull request merged**